platform/android/detect.py

```diff
@@ -52,9 +52,7 @@ def get_android_ndk_root():
 
 
 def get_flags():
-    return [
-        ("tools", False),
-    ]
+    return [("tools", False), ("use_volk", False)]
```

Comment:
```suggestion
    return [
        ("tools", False),
        ("use_volk", False),
    ]
```

Reply:
I was doing it, but it's rejected by static checks CI for some reason.

Reply:
It should work if you use a comma at the end.